ServerCore/Pages/Teams/Create.cshtml.cs

```diff
@@ -28,7 +28,12 @@ public async Task<IActionResult> OnGetAsync()
 
             if (EventRole != EventRole.play && EventRole != EventRole.admin)
             {
-                return NotFound();
+                return Forbid();
+            }
+
+            if (EventRole == EventRole.play && GetTeamId().Result != -1)
+            {
+                return Forbid();
```

Comment:
Should we do anything friendlier to let people know they're already on a team?

Reply:
Changed to NotFound with a message